### ISSUE TITLE
`Gitea` race condition for user creation after restart 

### ISSUE THREAD

**Issue by uwardlaw** [uwardlaw/vater#12]
Following a `clean.sh gitea`, `restart.sh` calls `setupGitea.py`

```
2021/10/27 14:25:05 main.go:117:main() [F] Failed to run app with [gitea admin user create --admin --username config --email fake@a.com --password config --must-change-password=false]: CreateUser: dial tcp 192.168.224.2:3306: connect: connection refused
```

**Comment by uwardlaw** [comment]
Very likely the same issue as #14

**Comment by uwardlaw** [comment]
Merged into #116
